law.cfg

```diff
@@ -28,6 +28,11 @@ inference_modules: columnflow.inference.example
 # wether or not the ensure_proxy decorator should be skipped, even if used by task's run methods
 skip_ensure_proxy: False
 
+# csv list of task families that inherit from ChunkedReaderMixin and whose output arrays should be
+# checked for non-finite values before saving them to disk (right now, supported tasks are
+# cf.CalibrateEvents, cf.SelectEvents, cf.ProduceColumns, cf.PrepareMLEvents, cf.MLEvaluation)
+check_finite_output: None
```

Comment:
Wondering if we should set this to (at least) `cf.SelectEvents` rather than `None` in the default law config. We can always change this back if it turns out that`nans` in a selection output are ever needed by an analysus, but IMO it doesn't hurt to be a bit more restrictive while we are still evaluating use cases. Up to you.

Reply:
In fact, I added all of them by default at first.
Then I noticed that a method in our bb𝜏𝜏 selection (a network that identifies Higgs b jets) actually uses nan values to mark certain cases ^^ Besides the fact that we should interpret theses cases anyway and store a different value on our end, maybe we should go with None, at least for now, to not assume that e.g. nan always means a bad thing by default.